sdk/communication/AzureCommunicationChat/Source/Registrar/RegistrarClient.swift

```diff
@@ -49,7 +49,7 @@ internal class RegistrarClient: PipelineClient {
         credential: CommunicationTokenCredential,
         registrationId: String,
         sessionConfiguration: URLSessionConfiguration? = nil,
-        authPolicy: Authenticating,
+        headersPolicy: HeadersPolicy,
         withOptions options: ClientOptions
```

Comment:
Does this work without an Authenticating policy? I thought the pipeline required one.